src/marqo/index.py

```diff
@@ -499,35 +501,37 @@ def verbosely_add_docs(i, docs):
                                  f'Allowed update_methods: ["replace", "update"] ')
             total_batch_time = timer() - t0
             num_docs = len(docs)
-            mq_logger.info(
-                f"   add_documents batch {i} roundtrip: took {(total_batch_time):.3f}s to add {num_docs} docs, "
-                f"for an average of {(total_batch_time/num_docs):.3f}s per doc.")
 
             if isinstance(res, list):
                 # with Server Batching (show processing time for each batch)
+                mq_logger.info(
+                    f"   add_documents batch {i} roundtrip: took {(total_batch_time):.3f}s to add {num_docs} docs, "
+                    f"for an average of {(total_batch_time / num_docs):.3f}s per doc.")
 
                 if isinstance(res[0], list):
                     # for multiprocess, timing messages should be arranged by process, then batch
                     for process in range(len(res)):
-                        mq_logger.info(f"       process {process}:")
+                        mq_logger.debug(f"       process {process}:")
 
                         for batch in range(len(res[process])):
                             server_batch_result_count = len(res[process][batch]["items"])
-                            mq_logger.info(f"           marqo server batch {batch}: "
+                            mq_logger.debug(f"           marqo server batch {batch}: "
                                             f"processed {server_batch_result_count} docs in {(res[process][batch]['processingTimeMs'] / 1000):.3f}s, "
                                             f"for an average of {(res[process][batch]['processingTimeMs'] / (1000 * server_batch_result_count)):.3f}s per doc.")
                 else:
                     # for single process, timing messages should be arranged by batch ONLY
                     for batch in range(len(res)):
                         server_batch_result_count = len(res[batch]["items"])
-                        mq_logger.info(f"       marqo server batch {batch}: "
+                        mq_logger.debug(f"       marqo server batch {batch}: "
                                         f"processed {server_batch_result_count} docs in {(res[batch]['processingTimeMs'] / 1000):.3f}s, "
                                         f"for an average of {(res[batch]['processingTimeMs'] / (1000 * server_batch_result_count)):.3f}s per doc.")
             else:
                 # no Server Batching
                 if 'processingTimeMs' in res:       # Only outputs log if response is non-empty
-                    mq_logger.info(f"   add_documents batch {i} Marqo processing: took {(res['processingTimeMs'] / 1000):.3f}s for Marqo to process & index {num_docs} "
-                                    f"docs (server unbatched), for an average of {(res['processingTimeMs'] / (1000 * num_docs)):.3f}s per doc.")
+                    mq_logger.info(
```

Comment:
if we only want 1 message to remain "info" per batch, do we want it to be this (processing time as reported by Marqo), or the one at line 502 (measured roundtrip time)?

Reply:
See my this comment: https://github.com/marqo-ai/py-marqo/pull/63/files/9d0b1c45c67df99089784a5cbb6ec114be7c90af#r1115317936